src/WattleScript.Interpreter/DataTypes/DataType.cs

```diff
@@ -68,7 +68,9 @@ public enum DataType
 		/// <summary>
 		/// A request to await a task
 		/// </summary>
-		AwaitRequest
+		AwaitRequest,
+		
+		Range
```

Comment:
XML comment needed

Reply:
Added in [ea86dbd](https://github.com/WattleScript/wattlescript/pull/36/commits/ea86dbd442ce1abc6a2f39118fec572d5ccebfb3)